### PR TITLE
[AddonManager] Add Python 2 Wb Awareness..

### PR DESCRIPTION
**Pull request by Syres916**
...and improve Git binary version handling.

See https://forum.freecadweb.org/viewtopic.php?f=10&t=37394&start=70#p393876

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/AddonManager/addonmanager_workers.py [hunk 1]
```diff
@@ -130,6 +140,27 @@ def run(self):
             for repo in repos:
                 self.addon_repo.emit(repo)
             self.info_label.emit(translate("AddonsInstaller", "Workbenches list was updated."))
+        git_exe = utils.checkGitBinary()
+        import platform
```
Comment (luzpaz) [inline src/Mod/AddonManager/addonmanager_workers.py:144]: This is already mentioned on line 323  
https://github.com/FreeCAD/FreeCAD/pull/3420/files#diff-86151fd926d4afd8c19a364370c576f0R323

Comment (Syres916) [reply src/Mod/AddonManager/addonmanager_workers.py:144]: If I comment out either of the import platform lines then errors are raised. I have noticed that I can remove a couple of import re statements which I will correct.

**Comment by luzpaz** [comment]
I think also add @microelly2's [reconstruction wb](https://github.com/microelly2/reconstruction/) since it's python2 based as well

**Comment by vocx-fc** [comment]
Maybe check this list as well, [Python3 and Qt5 Compatible Addon Workbenches (in preparation for 0.18)](https://forum.freecadweb.org/viewtopic.php?f=10&t=30624).

**Comment by yorikvanhavre** [comment]
Tell me if you prefer to merge it now or add commits @Syres916

**Comment by Syres916** [comment]
@yorikvanhavre please merge it and I'll submit a further PR when I get time to go through the list, thanks.

**Comment by yorikvanhavre** [comment]
Here we go!

**Pull request merged**